circle.yml

```diff
@@ -11,9 +11,11 @@ machine:
 dependencies:
   pre:
     # we want firefox nightly in order to install an unsigned screenshots addon
-    - pip install mozdownload mozinstall
-    - mozdownload --version latest --type daily --destination firefox.tar.bz2
-    - mozinstall firefox.tar.bz2
+    # - pip install mozdownload mozinstall
```

Comment:
If we put in `pip install requests` it would probably fix load_test_excercise.  I'm guessing one of these packages themselves required requests.